content/cloud/github-actions.md

```diff
@@ -0,0 +1,190 @@
+title = "Deploying Spin Apps Using GitHub Action"
+template = "cloud_main"
+date = "2023-03-27T21:00:01Z"
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/github-actions.md"
+
+---
+- [Deploying Spin Apps With Github Actions](#deploying-spin-apps-with-github-actions)
+  - [Prerequisites](#prerequisites)
+  - [Creating a New Github Repository](#creating-a-new-github-repository)
+  - [Creating a New Spin Application](#creating-a-new-spin-application)
+  - [Setting Up Github Actions](#setting-up-github-actions)
+  - [Configuring GitHub Actions](#configuring-github-actions)
+  - [Set a Github Repository Secret With Your Fermyon Cloud Personal Access Token (PAT)](#set-a-github-repository-secret-with-your-fermyon-cloud-personal-access-token-pat)
+  - [Push Changes to Your Registry to Deploy Spin App](#push-changes-to-your-registry-to-deploy-spin-app)
+- [Next Steps](#next-steps)
+
+## Deploying Spin Apps With Github Actions
+
+GitHub Action is a Continuous Integration and Continuous Deployment (CI/CD) platform for GitHub developers. With GitHub Actions, developers can create workflows that build, test and even deploy (to production) pull requests that have been merged into their repositories. There are thousands of GitHub Actions, published by companies and individual enthusiasts alike, that automate otherwise mundane, repetitive in-house development tasks.
+
+Fermyon's Spin uses three GitHub Actions. For example:
+* `fermyon/actions/spin/setup` - enables users to install the Spin CLI and necessary plugins,
+* `fermyon/actions/spin/push` - enables users to push their Spin applications to an OCI registry, and
+* `fermyon/actions/spin/deploy` - enables users to deploy their Spin applications to Fermyon Cloud.
+
+This tutorial will focus specifically on `fermyon/actions/spin/deploy`. Upon completing this tutorial, you should have a GitHub repository that builds and deploys a Spin application to Fermyon Cloud every time you commit a pull request. Let's get started!
```

Comment:
everytime you `merge` (instead of commit) a pull request?

Reply:
Thanks @rajatjindal
Updating this in the next push to Kenzie's branch. Thank you!